armory/scenarios/configs/cifar10_baseline.json

```diff
@@ -0,0 +1,50 @@
+{
+    "adhoc": null,
+    "attack": {
+        "budget": {
+            "norm": [
+                "Linf"
+            ]
+        },
+        "knowledge": "white",
+        "kwargs": {
+            "eps": 0.2
+        },
+        "module": "art.attacks",
+        "name": "FastGradientMethod"
+    },
+    "dataset": {
+        "batch_size": 16,
+        "module": "armory.data.datasets",
+        "name": "cifar10"
+    },
+    "defense": {
+        "kwargs": {},
+        "module": "armory.art_experimental.defences.madry_trainer_generator",
+        "name": "AdversarialTrainerMadryPGD"
+    },
+    "evaluation": {
+        "eval_file": "armory.scenarios.base"
+    },
+    "metric": null,
+    "model": {
+        "fit_kwargs": {
+            "nb_epochs": 3
+        },
+        "model_kwargs": {},
+        "module": "armory.baseline_models.keras.cifar",
+        "name": "get_art_model",
+        "weights_file": null,
+        "wrapper_kwargs": {}
+    },
+    "scenario": {
+        "kwargs": {},
+        "module": "armory.scenarios.cifar10",
+        "name": "Cifar10"
+    },
```

Comment:
A little worried about how this will be handled for configs without a scenario. Perhaps we take this time to enforce evaluations always be ran on a scenario (new scenarios could obviously be added by endusers).

Reply:
Agreed. Will standardize when we have all of the scenarios incorporated.